content/_ext/solutionpage.rb

```diff
@@ -7,10 +7,11 @@
 class SolutionPage
   def execute(site)
     solutions = site.pages.select { |p| p.source_path =~ /solutions/ }
-
+    print "Loading solution pages\n"
```

Comment:
For what it's worth, people generally use `puts` in Ruby which will append a newline, it's roughly equivalent to `System.out.println` vs. `print`